cluster/log-dump/log-dump.sh

```diff
@@ -282,7 +282,7 @@ function dump_nodes_with_logexporter() {
   sed -i'' -e "s@{{.EnableHollowNodeLogs}}@${enable_hollow_node_logs}@g" "${KUBE_ROOT}/cluster/log-dump/logexporter-daemonset.yaml"
 
   # Create the logexporter namespace, service-account secret and the logexporter daemonset within that namespace.
-  KUBECTL="${KUBECTL:-${KUBE_ROOT}/cluster/kubectl.sh}"
+  KUBECTL="${KUBE_ROOT}/cluster/kubectl.sh"
```

Comment:
Sorry, I don't understand. Yes, I know what KUBECT can be set to. But how is that problematic?